README.txt

```diff
@@ -1,13 +1,19 @@
-Installer: http://MrFeinberg.com/haikufinder-1.1.zip
+Files: http://MrFeinberg.com/haikufinder-1.1.zip or use github repo.
 
-* 
-*	REQUIRES NLTK, THE NATURAL LANGUAGE TOOLKIT
-*	http://www.nltk.org/download
-*
+Installation requires NLTK and its punkt package. First-time steps
+Unzip main file.
+pip install nltk
+python
```

Comment:
you can combine these ntlk steps as:
```
python -m nltk.downloader punkt
```

I'm still investigating the right way to do this in `setup.py`.